tools/test-oidc-flow.sh

```
@@ -0,0 +1,468 @@
+}
+
+#######################################
+# Step 0: Check Signal K OIDC status
+#######################################
+log "Step 0: Checking Signal K OIDC status..."
+
+LOGIN_STATUS=$(curl "${CURL_OPTS[@]}" "$SIGNALK_URL/skServer/loginStatus")
+echo "$LOGIN_STATUS" > step0_login_status.json
+
+OIDC_ENABLED=$(echo "$LOGIN_STATUS" | python3 -c "import sys,json; print(json.load(sys.stdin).get('oidcEnabled', False))" 2>/dev/null || echo "false")
+OIDC_LOGIN_URL=$(echo "$LOGIN_STATUS" | python3 -c "import sys,json; print(json.load(sys.stdin).get('oidcLoginUrl', ''))" 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] OIDC_LOGIN_URL appears unused. Verify use (or export if used externally).

(SC2034)